README.md

```diff
@@ -833,27 +833,66 @@ The output of this command is a ready-to-use Kogito Operator image that you can
 
 ### Deploying to OpenShift 4.x for development purposes
 
-To install the Kogito Operator on OpenShift 4.x for end-to-end (E2E) testing, ensure that you have access to a `quay.io` account to create an application repository. Follow the Operator Courier [authentication](https://github.com/operator-framework/operator-courier/#authentication) instructions to obtain an account token. This token is in the format `basic XXXXXXXXX` and both words are required for the command.
+To install the Kogito Operator on OpenShift 4.x for end-to-end (E2E) testing, ensure that you have access to a `quay.io` 
+account to create an application repository. 
 
-Push the Operator bundle to your quay application repository as shown in the following example:
+Follow the steps below:
+
+1. Run `make prepare-olm version=0.7.0`
```

Comment:
This does not work, need to be `make prepare-olm version=0.7.0-rc4` or ?

Reply:
btw, testing this I get this error:
```
make[1]: Leaving directory '/home/tradisso/projects/kogito/kogito-cloud-operator'
ERROR: channel.currentCSV kogito-operator.v0.6.0 is not included in list of csvs [operatorhub/kogito-operator.package.yaml]
Resulting bundle is invalid, input yaml is improperly defined.
make: *** [Makefile:99: prepare-olm] Error 1
```

Reply:
This is just a README that will be changed back to 0.7.0. But anyway, I’ll change it